Jenkinsfile

```
@@ -182,11 +181,7 @@ stage('Build') {
                                 // Publication is enabled by default, disabled when simulating a LTS
                                 if (env.PUBLISH == 'true') {
                                     infra.withDockerCredentials {
-                                        sh '''
-                                        docker buildx create --use
-                                        docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
-                                        make publish
-                                        '''
+                                        sh 'make docker-init publish'
```

```suggestion
                                        sh 'make docker-init'
                                        sh 'make publish'
```

As GNU `make` is built to parallelize as much as possible and there are no constraints expressed between the `docker-init` and `publish` targets, do you mind splitting in 2 instructions to avoid any risk of concurent executions (it is publication so hard to replay or reproduce so I prefer playing it safe).

My suggestion proposes to use 2 distinct `sh` steps, so it's easier for us to navigate the pipeline and evaluate visually the execution timings or access logs. The second `sh` step only had a few 100s ms to execute the second SH script so it is more than acceptable in this context.